.travis.yml

```diff
@@ -1,32 +1,43 @@
-# Test build the MoveIt tutorials. Author: Dave Coleman
-sudo: required
-dist: xenial
-language: ruby
-rvm:
-  - 2.7
-python:
-  - "2.7"
+# This config file for Travis CI utilizes https://github.com/ros-planning/moveit_ci/ package.
+os: linux
+dist: bionic  # distro used by Travis, moveit_ci uses the docker image's distro
+services:
+  - docker
+language: cpp
+cache: ccache
+compiler: gcc
+
+notifications:
+  email: true
 
 env:
   global:
-    - SCRIPT=.moveit_ci/travis.sh
-    - ROS_DISTRO=noetic
-    - ROS_REPO=ros-testing
+    - MOVEIT_CI_TRAVIS_TIMEOUT=85  # Travis grants us 90 min, but we add a safety margin of 5 min
+    - ROS_DISTRO=foxy
+    - ROS_REPO=ros
+    - UPSTREAM_WORKSPACE=moveit2_tutorials.repos
```

Comment:
So I had to add all of `moveit2.repos` to this file. Is there a way to have 2 repos files here and pull down source from both files?

Reply:
I think we support multiple repos files, adding multiple comma-separated entries in quotes should work: `"1.repos, 2.repos "`